css/style.css

```diff
@@ -1,50 +1,275 @@
 /*
 Project Name: Blank Template
 Client: Your Client
-Author: Your Name
+Author: Iko Gabay
 Developer @GA in NYC
 */
 
 /******************************************
 /* SETUP
 /*******************************************/
 
+
+
+
+
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
 html {
   box-sizing: border-box;
 }
 
 *,
 *::before,
 *::after {
-  box-sizing: inherit;
+  box-sizing: inherit
 }
 
-/******************************************
-/* BASE STYLES
-/*******************************************/
 
 body {
-  color: #000;
-  font-size: 12px;
+  color: #1a1a1a;
+  font-size: 26px;
   line-height: 1.4;
-  font-family: Helvetica, Arial, sans-serif;
+  font-family: 'Open Sans'
+  background-color: #f0efef;
+}
+
+a {
+  text-decoration: none;
 }
 
+
 /******************************************
 /* LAYOUT
 /*******************************************/
 
 /* Center the container */
 
-.container {
+
+header {
+  height: 120px;
+  background: #131e27;
+  padding-top: 20px;
   width: 100%;
   margin: auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0px 40px;
+  font-family: 'Open Sans', sans-serif;
 }
 
-header {}
+.navigation a {
+  color: #f9e42e;
+  font-weight: 600;
+  text-decoration: none;
+  font-size: 24px;
+}
 
-footer {}
+.navigation a:hover {
+  color: #FFFFFF;
+  font-weight: 600;
+  text-decoration: none;
+  font-size: 24px;
+}
+.navigation ul {
+  display: flex;
+  justify-content: flex-end;
+  flex-direction: row;
+  padding: 40px;
+}
 
-/******************************************
-/* ADDITIONAL STYLES
-/*******************************************/
\ No newline at end of file
+.navigation li {
+  padding-right: 40px;
+  list-style: none;
+}
+
+.navigation h1 {
+  font-weight: 800;
+  font-size: 46px;
+  color: #FFFFFF;
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: row;
+}
+
+.content {
+  display: flex;
+  justify-content: space-between;
+  flex-grow: 1;
+  align-items: baseline;
+  font-family: 'Open Sans', sans-serif;
+}
+
+.blog {
+  width: 60%;
+  margin-top: 80px;
+  margin-left: 40px;
+  display: flex;
+  flex-direction: column;
+
+}
+
+.blog article {
+  border-bottom: 1px solid #043049;
+  margin-bottom: 40px;
+}
+
+.blog h2 {
+  font-size: 40px;
+  font-weight: 600;
+  line-height: 1.3;
+  color: #043049;
+  padding-bottom: 30px;
+
+}
+
+.blog img {
+  padding-bottom: 40px;
+  width: 700px;
+  height: auto;
+}
+
+.blog p {
+  color: #616161;
+  font-size: 24px;
+  padding-bottom: 30px;
+}
+
+.dropcap {
+  float: left;
+  color: #616161;
+  font-size: 60px;
+  font-weight: 600;
+  line-height: 60px;
+  padding:8px 8px 5px 0;
+}
```

Comment:
Nice work on the drop cap.

Another way you could have done this would be to use a pseudo selector:

```html
<p>Lorem ipsum ... </p>
```

```css
p:first-letter {
  /* your css styles for dropcap go here */
}
```